### PR TITLE
Import install plans

### PR DESCRIPTION
**Pull request by polfliet**
Based on [this list of installPlans](https://docs.google.com/spreadsheets/d/1qlQoXwRd7fXffIGswZxa6dQWNuzuGvF9682ArFniizQ/edit#gid=171035041) I created a script to import them to the quickstarts.

This upgrades many documentation packs that can actually reference an existing installPlan. For example like all of the quickstarts that are related to the Java APM agent, they will now reference the installPlan for the Java agent

### CODE REVIEW

**Comment by polfliet** [comment]
@jbeveland27 would love your review on this since you have been working on this

**Comment by jpvajda** [comment]
since this PR came in from a fork, the add PR support labels and Generate UUIDs checks are failing, that's a known problem we are looking into, but this PR should be ok to merge once @jbeveland27 has time to 👍  it up.

File: packs/datastax-cassandra/config.yml [hunk 1]
```diff
@@ -16,3 +16,6 @@ documentation:
 keywords:
   - infrastructure
   - database
+installPlans:
+  - infra-agent-targeted
+  - cassandra-integration
```
Comment (jbeveland27) [inline packs/datastax-cassandra/config.yml:21]: This quickstart has a few things incorrect:

- It's not an on host integration
- It should be nested under the `java` quickstart
- The config needs to be reworked a bit to reflect that it's an APM agent install (fix `keywords`, etc.)

```suggestion
installPlans:
  - setup-java-agent
```

File: packs/java/jayrock/config.yml [hunk 2]
```diff
@@ -22,3 +22,5 @@ documentation:
 keywords:
   - apm
   - java
+installPlans:
+  - setup-java-agent
```
Comment (jbeveland27) [inline packs/java/jayrock/config.yml:26]:
```suggestion
installPlans:
  - dotnet-agent
```

File: packs/java/jayrock/config.yml [hunk 3]
```diff
@@ -22,3 +22,5 @@ documentation:
 keywords:
   - apm
   - java
+installPlans:
+  - dotnet-agent
```
Comment (jbeveland27) [inline packs/java/jayrock/config.yml:26]: this pack should be moved to the `dotnet` folder

File: packs/perfmon/config.yml [hunk 4]
```diff
@@ -20,3 +20,5 @@ documentation:
     url: https://docs.newrelic.com/docs/perfmon-open-source-integration
 keywords:
   - windows
+installPlans:
+  - guided-install
```
Comment (jbeveland27) [inline packs/perfmon/config.yml:24]: This is an integration that's not supported by guided install
```suggestion
```

File: packs/unix/config.yml [hunk 5]
```diff
@@ -22,3 +22,5 @@ documentation:
 keywords:
   - os
   - operating system
+installPlans:
+  - guided-install
```
Comment (jbeveland27) [inline packs/unix/config.yml:26]: This is an integration that's not supported by guided install
```suggestion
```

**Review by jbeveland27 — CHANGES_REQUESTED** [review]
Left some feedback.

**Comment by jbeveland27** [comment]
Made a few adjustments to fix install plans that had easy changes. There are a few packs that need addressed in an additional PR:

* `datastax-cassandra` - this pack should be moved under the java agent. It's not a standalone on-host integration
* `golang/go-insights` - this pack should probably just be removed, it's not the Golang agent, it's a separate module written 
* by the DTK team and it's been deprecated
* `java/grpc` - grpc is more than just the Java Agent, we should rework this pack to be more accurate
* `java/jayrock` - this is a dotnet pack, not a java pack. Should move under the `dotnet` directory
* `mysql` - this has reference to MariaDB in the name, but there's a standalone mariadb pack. Should fix the name
* php/sqlite - seems odd that this is our only sqlite reference. Should SQLite be its own pack?

----

These packs should be removed IMO. These really just point to the python agent docs, but it doesn't feel accurate to say we're doing anything special to monitor these. Our docs just link to these project sites (and some are 404s).
* python/dropbox
* python/pywapi
* python/urllib
* python/urllib2
* python/urllib3

image

**Review by jbeveland27 — APPROVED** [review]

**Comment by jbeveland27** [comment]
#231 addresses all but: 

* `java/grpc` - grpc is more than just the Java Agent, we should rework this pack to be more accurate
* `php/sqlite` - seems odd that this is our only sqlite reference. Should SQLite be its own pack?

**Pull request merged**